openstax/middleware.py

```diff
@@ -195,6 +195,8 @@ def page_by_slug(self, page_slug):
             return Supporters.objects.all()
         if page_slug == 'openstax-homepage':
```

Comment:
can this section be deleted?

Reply:
Not if we want this to work before deployment of FE part 2.

This should be removed as part of the migration of the home page to the top level of the content root.